lib/functions-utility.php

```diff
@@ -7,6 +7,7 @@ function url_get_contents($Url) {
   }
 
   $ch = curl_init();
+  curl_setopt($ch, CURLOPT_SSL_VERIFYPEER, false);
```

Comment:
I guess this is only sketchy if we use the function wrong ourselves. So it is ok.